wave_utils/CMakeLists.txt

```diff
@@ -7,6 +7,7 @@ FIND_PACKAGE(Eigen3 REQUIRED)
 
 # INCLUDES
 INCLUDE_DIRECTORIES(/usr/include/eigen3)
```

Comment:
I saw from Issue #45 that we were looking to modify these hardcoded directories too. For a separate issue, I guess?